dkc/core/rest/file.py

```diff
@@ -83,11 +101,39 @@ def get_serializer_class(self):
             return FileUpdateSerializer
         return FileSerializer
 
+    def _validate_authorized_upload(self, authorization: str, folder: Folder) -> User:
+        try:
+            upload_id = Signer().unsign(authorization)
+        except BadSignature:
+            logger = logging.getLogger('django.security.SuspiciousOperation')
+            logger.exception('Authorized upload signature tampering detected')
```

Comment:
Should this be a `logging.warning` instead?

Reply:
Verdict: yes. It will log stack trace and locals.